apps/frontend/src/utils/helperFunctions.tsx

```diff
@@ -36,13 +36,16 @@ export const getUniqueArray = <T,>(array: (T | null)[]) =>
     return value !== null && self.indexOf(value) === index;
   });
 
-export const setSortDirectionOnSortColumn = (
+export const setSortDirectionOnSortField = (
   columns: Column<any>[],
-  sortColumn: number | null | undefined,
+  sortField: string | null | undefined,
   sortDirection: string | null | undefined
 ) => {
-  if (sortColumn !== undefined && sortColumn !== null && sortDirection) {
-    columns[sortColumn].defaultSort = sortDirection as SortDirectionType;
+  if (sortField !== undefined && sortField !== null && sortDirection) {
```

Comment:
Does not if(sortField) check for both null and undefined.

Reply:
It might be but I was thinking that we want to support empty strings like '' which will return false in this case.